packages/template-retail-react-app/app/components/search/index.test.js

```diff
@@ -45,7 +78,7 @@ test('renders SearchInput', () => {
 })
 
 test('changes url when enter is pressed', async () => {
-    const user = userEvent.setup()
+    const user = setupUserEvent()
 
     renderWithProviders(<SearchInput />, {
```

Comment:
Please fix for all the tests
```suggestion
   const {user} = renderWithProviders(<SearchInput />, {}
```

Reply:
@alexvuong my observations is this project does prefer ``` const {user} = renderWithProviders(...)``` in most cases as per your suggestion. However, this specific test file is an exception - It needs fake timers with custom advanceTimers configuration. The ```renderWithProviders``` function doesn't support custom user options and it always uses userEvent.setup() without options. Which means we end up creating a custom user event configuration. Which we have to put that out off the scope off this work and create a work item in our backlog to tackle it next.